src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -19,10 +19,19 @@ import { IQuery } from '../../rest/Query';
 import { IQueryResults } from '../../rest/QueryResults';
 
 export interface IQuerySummaryOptions {
-  enableSearchTips?: boolean;
```

Comment:
This variable still exists in the code, it just have been moved for consistency with the layout of the options.